sdg/edges.py

```diff
@@ -116,13 +146,10 @@ def prune_grand_parents(edges):
 
 def edge_detection(inid, df):
     """Check dependencies between columns and write out the edges
-
```

Comment:
Any particular reason to remove all the blank lines?

Reply:
I don't know why that happened - I usually like plenty of space around my comments, so I don't think it was intentional. I might have changed the file format a few times while finding the best way to edit it, which could have removed the blank lines.